website/blog/2024-09-01-model-as-code-logging/index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Model-as-Code Logging in MLflow for Enhanced Model Management
+tags: [pyfunc, genai]
+slug: model_as_code
+authors: [awadelrahman-ahmed, daniel-liden]
+thumbnail: img/blog/thumbnail_model_as_code.gif
+---
+
+We all—well, most of us—remember November 2022 when the public release of ChatGPT by OpenAI marked a significant turning point in the world of AI. While GenAI had been evolving for some time, ChatGPT, built on OpenAI's GPT-3.5 architecture, quickly captured the public’s imagination. This led to an explosion of interest in GenAI, both within the tech industry and among the general public.
+
+On the tools side, MLflow continues to solidify its position as the favorite tool for MLOps among the ML community. However, the rise of GenAI has introduced new needs in how we use MLflow. One of these new challenges is how we log model artifacts in MLflow. If you’ve used MLflow before (and we bet you have), you’re probably familiar with the `mlflow.log_model()` function and how it efficiently **pickles** model artifacts (I’ve bolded "pickles" because it’s a key term in this post).
+
+Particularly with GenAI, there’s a new requirement: logging the model "as code," instead of serializing it into a pickle file. And guess what? This need isn’t limited to GenAI models. So, in this post we will explore this concept and how MLflow has adapted to meet this new requirment.
```

Comment:
Can we convert the reference of `models from code` to a link to https://mlflow.org/docs/latest/models.html#models-from-code ?

Reply:
please check if this is what you meant!